src/GamingDeviceButton.js

```diff
@@ -0,0 +1,35 @@
+export class GamingDeviceButton extends HTMLButtonElement {
+  constructor(buttonNumber) {
+    super();
+
+    // set default attributes
+    this._pressed = false;
+    this._touched = false;
+    this._value = 0.0;
+
+    this.innerHTML = `button ${buttonNumber + 1}`;
+    this.type = "button";
+
+    this.addEventListener("click", () => {
+      // connect the button's gamepad
+      const gamingDevice = event.target.parentElement.parentElement;
```

Comment:
hmmm.. this seems a bit fragile. Maybe we should have a specific button that is the "on" button. However, that could be designated in the gaming device code (just as an example):

```JS
// in GamingDevice, let's pretend button 18 is the on button
const theOnButton = new GamingDeviceButton(18);

theOnButton.addEventListener("click", () => {
    // "this" would be gaming device
    this.isConnected = !this.isConnected; // toggle on and off
})
```